examples/src/repeated_compute/mod.rs

```diff
@@ -109,7 +109,10 @@ async fn compute(local_buffer: &mut [u32], context: &WgpuContext) {
     // One of those can be calling `Device::poll`. This isn't necessary on the web as devices
     // are polled automatically but natively, we need to make sure this happens manually.
     // `Maintain::Wait` will cause the thread to wait on native but not the web.
```

Comment:
since you nicely updated it elsewhere

```suggestion
    // `Maintain::Wait` will cause the thread to wait on native but not on WebGpu.
```